README.md

```diff
@@ -1,106 +1,112 @@
+[![Requires.io]](https://requires.io/github/ITISFoundation/osparc-simcore/requirements/?branch=master "State of third party python dependencies")
+[![travis-ci]](https://travis-ci.org/ITISFoundation/osparc-simcore "State of CI: build, test and pushing images")
+[![coverals.io]](https://coveralls.io/github/ITISFoundation/osparc-simcore?branch=master "Test coverage")
+[![github.io]](https://itisfoundation.github.io/)
+[![itis.dockerhub]](https://hub.docker.com/u/itisfoundation)
+
+
+<!-- THIS SERVICE MALFUNCTIONS
+[![webserver]](https://microbadger.com/images/itisfoundation/webserver "More on itisfoundation/webserver:staging-latest image")
+[![director]](https://microbadger.com/images/itisfoundation/director "More on itisfoundation/director:staging-latest image")
+[![sidecar]](https://microbadger.com/images/itisfoundation/sidecar "More on itisfoundation/sidecar:staging-latest image")
+[![storage]](https://microbadger.com/images/itisfoundation/storage "More on itisfoundation/storage:staging-latest image")
+-->
+
+
+<!-- ADD HERE ALL BADGE URLS -->
+[Requires.io]:https://img.shields.io/requires/github/ITISFoundation/osparc-simcore.svg
+[travis-ci]:https://travis-ci.org/ITISFoundation/osparc-simcore.svg?branch=master
+[coverals.io]:https://coveralls.io/repos/github/ITISFoundation/osparc-simcore/badge.svg?branch=master
+[github.io]:https://img.shields.io/website-up-down-green-red/https/itisfoundation.github.io.svg?label=documentation
+[itis.dockerhub]:https://img.shields.io/website/https/hub.docker.com/u/itisfoundation.svg?down_color=red&label=dockerhub%20repos&up_color=green
+[webserver]:https://img.shields.io/microbadger/image-size/itisfoundation/webserver/staging-latest.svg?label=webserver&style=flat
+[director]:https://img.shields.io/microbadger/image-size/itisfoundation/director/staging-latest.svg?label=director&style=flat
+[sidecar]:https://img.shields.io/microbadger/image-size/itisfoundation/sidecar/staging-latest.svg?label=sidecar&style=flat
+[storage]:https://img.shields.io/microbadger/image-size/itisfoundation/storage/staging-latest.svg?label=storage&style=flat
+<!---------------------------->
+
+## Overview
+
+![service-web](docs/img/service-interaction.svg)
+
+
+A graph view of services, ports, volumes and service dependencies:
+![](docs/img/docker-compose-wo-networks.png)
+
+## Contributing
+
+Would you like to make a change or add something new? Please read the [contributing guidelines](CONTRIBUTING.md).
+## Usage
+
+```bash
+  # clone repo
+  git clone https://github.com/ITISFoundation/osparc-simcore.git
+  cd osparc-simcore
+
+  # show setup info and build core services
+  make info build
+
+  # starts swarm and deploys services
+  make up-prod
+
+  # display swarm configuration
+  make info-swarm
+
+  # open browser in:
+  #  localhost:9081 - simcore front-end site
+  #
+  xdg-open http://localhost:9081/
+
+  # stops
+  make down
+```
+
+## Requirements
+
+To build and run:
+
+- docker
```

Comment:
for docker, python etc ... , can you perhaps add a reference to the CI where we specify exactly which versions we use?